vndr.cfg

```diff
@@ -0,0 +1,11 @@
+github.com/Sirupsen/logrus 26709e2714106fb8ad40b773b711ebce25b78914
+github.com/coreos/go-systemd v4
+github.com/docker/docker 0f5c9d301b9b1cca66b3ea0f9dec3b5317d3686d
+github.com/docker/go-units v0.1.0
+github.com/godbus/dbus v3
+github.com/golang/protobuf f7137ae6b19afbfd61a94b746fda3b3fe0491874
+github.com/opencontainers/runtime-spec v1.0.0-rc1
+github.com/seccomp/libseccomp-golang 32f571b70023028bd57d9288c20efbcb237f3ce0
+github.com/syndtr/gocapability 2c00daeb6c3b45114c80ac44119e7b8801fdd852
+github.com/urfave/cli edb24d02aa3cea2319c33f2836d4a5133907fe4c
```

Comment:
Think we could upgrade this to `v1.18.0`? :pray: :innocent: (https://github.com/urfave/cli/commit/edb24d02aa3cea2319c33f2836d4a5133907fe4c)